src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx

```diff
@@ -80,6 +80,29 @@ export class ContactInformationEditView extends Component {
     }
   }
 
+  focusOnUpdateButton() {
+    const focusableElement = this.editForm?.querySelector(
+      '[data-testid="save-edit-button"]',
+    );
+    if (focusableElement) {
+      focusableElement.focus();
+    }
+  }
+
+  focusOnError() {
+    const focusableElement = this.errorAlert?.querySelector(
```

Comment:
same here. I think `focusElement('button[aria-label="Close notification"]')` should work